### ISSUE TITLE
Support scrolling by changing the portion of the framebuffer read by the display

### ISSUE THREAD

**Issue by tannewt** [adafruit/circuitpython#1170]
This is particularly helpful for scrolling text output so that the new line only needs to be transferred.

The gameboy uses this technique. Its explained here: https://youtu.be/HyzD8pNlpwI?t=31m43s

**Comment by tannewt** [comment]
Really? The PTLAR only affects new pixels? I haven't tested it yet though.

**Comment by deshipu** [comment]
Yes, the datasheet is not clear about this, but it controls how the pixels are written by the memory write commands, not how they are sent to the display. At least that's what I figured out with my experiments, I might remember it wrong, it was a year ago.

**Comment by deshipu** [comment]
Actually, that wasn't PTLAR. PTLAR is for only using a part of the display (for example, when the rest is covered by the bezel).
